src/app/(admin)/admin-dashboard/courses/page.tsx

```
@@ -0,0 +1,7 @@
+import React from "react";
+
+type Props = {};
+
+export default function Page({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)